pubspec.yaml

```diff
@@ -1,14 +1,15 @@
 name: flutter_dotenv
-version: 3.1.0
+version: 3.2.0-nullsafety.0
```

Comment:
This should be 4.0.0 because of breaking change.

Reply:
Updated!